### PR TITLE
refactor: change some SessionManager variable names for consistency

### PR DESCRIPTION
**Pull request by elliotfontaine**
Got weirded out by `load_session()` only returning the token 😆

### CODE REVIEW

File: addons/talo/apis/players_api.gd [hunk 1]
```diff
@@ -28,7 +28,7 @@ func identify_steam(ticket: String, identity: String = "") -> void:
 
 ## Flush and sync the player's current data with Talo.
 func update() -> void:
-	var res = await client.make_request(HTTPClient.METHOD_PATCH, "/%s" % Talo.current_player.id, { props = Talo.current_player.get_serialized_props() })
+	var res = await client.make_request(HTTPClient.METHOD_PATCH, "/%s" % Talo.current_player.id, {props = Talo.current_player.get_serialized_props()})
```
Comment (tudddorrr) [inline addons/talo/apis/players_api.gd:31]: would prefer to keep these spaces for readability

**Review by tudddorrr — CHANGES_REQUESTED** [review]
just the one change otherwise lgtm thanks!

File: addons/talo/apis/players_api.gd (hunk 1 above)
Comment (elliotfontaine) [reply addons/talo/apis/players_api.gd]: Sorry that was my GDScript formatter on VSCode !

**Review by tudddorrr — APPROVED** [review]
thanks!

**Pull request merged**